backend/api/views/quiz.py

```diff
@@ -1,80 +1,77 @@
 from django.http import Http404
-from rest_framework.views import APIView
-from rest_framework.request import Request
-from rest_framework import status
 from pymongo.collection import ReturnDocument
+from rest_framework import status
+from rest_framework.request import Request
+from rest_framework.views import APIView
```

Comment:
Alphabetic order 😭 And they dare call me a perfectionnist 😂 
(added to my list of best practices)

Reply:
haha. Don't mess with order